cli/command/stack/remove.go

```diff
@@ -39,7 +39,17 @@ func runRemove(dockerCli command.Cli, opts removeOptions) error {
 	client := dockerCli.Client()
 	ctx := context.Background()
 
-	var errs []string
+	var (
+		errs    []string
+		secrets []swarm.Secret
+		configs []swarm.Config
+	)
```

Comment:
Updated; was thinking that declaring the variables inside the loop was bad, but reading up, Go is smart enough to handle that 👍